features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -552,6 +552,11 @@
         <DCRM>
             <ApplicationRolePermissionRequiredToView>{{oauth.dcrm.application_role_permission_required_to_view}}</ApplicationRolePermissionRequiredToView>
         </DCRM>
+
+        <!--
+        By enabling this config, additional parameters will be append to OAuth2 error url
+        -->
+        <AllowAdditionalParamsFromErrorUrl>false</AllowAdditionalParamsFromErrorUrl>
```

Comment:
We can comment this config as from the code level we are properly handling the default

Reply:
done